src/pages/Transaction/TransactionCellScript/index.tsx

```diff
@@ -333,8 +333,24 @@ export default ({ cell, onClose, txStatus }: { cell: State.Cell; onClose: Functi
           <div className="transaction__detail_copy">
             <TransactionDetailCopyButton onClick={onClickCopy}>
               <div>{i18n.t('common.copy')}</div>
-              <img src={isMainnet() ? CopyIcon : CopyBlueIcon} alt="copy" />
+              <CopyIcon />
             </TransactionDetailCopyButton>
+            {(state === CellState.LOCK || state === CellState.TYPE) &&
+              content &&
+              (content as State.Script).codeHash &&
+              (content as State.Script).hashType && (
+                <TransactionDetailScriptButton
+                  onClick={() => {
+                    window.open(
+                      `/script/${(content as State.Script).codeHash}/${(content as State.Script).hashType}`,
+                      '_blank',
+                    )
+                  }}
```

Comment:
I see `TransactionDetailScriptButton` is a new styled components, why not just based on `styled.a` to create, but use `styled.div + onClick + window.open`?

Reply:
> I see `TransactionDetailScriptButton` is a new styled components, why not just based on `styled.a` to create, but use `styled.div + onClick + window.open`?

changed to `styled.a`